docs/farming-&-staking/staking/intro.md

```diff
@@ -40,32 +40,23 @@ Subspace introduces the Decoupled Execution Framework (DecEx) to tackle the stat
 
 ### Operator hardware requirements
 
-:::note
-The hardware requirements have not been benchmarked, and these are our best estimates. We would appreciate your feedback if you feel that the requirements listed here are too high or too low. 
-:::
-
-:::tip
-Our suggested specs are not necessarily applicable to Stake Wars. We encourage all interested participants to join Stake Wars, even if your hardware does not meet the listed requirements. 
-:::
-
 **CPU**: 
 - x86-64 compatible;
 - Intel Ice Lake, or newer (Xeon or Core series); AMD Zen3, or newer (EPYC or Ryzen);
 - 4 physical cores @ 3.4GHz;
-- Simultaneous multithreading disabled (Hyper-Threading on Intel, SMT on AMD);
 - Prefer single-threaded performance over higher cores count. A comparison of single-threaded performance can be found [here](https://www.cpubenchmark.net/singleThread.html).
 
 **Storage**:
 - An NVMe SSD of 1 TB. In general, the latency is more important than the throughput.
 
 **Memory**:
-- 32 GB DDR4 ECC.
+- 16 GB.
 
 **System**:
 - Linux Kernel 5.16 or newer.
 
 **Network**:
-- The minimum symmetric networking speed is set to 500 Mbit/s.
+- The minimum symmetric networking speed is set to 50 Mbit/s.
```

Comment:
One of my concerns about running an operator was the network usage.  500 is a massive amount of bandwidth to allocate to one app.  I only have 300 down and 30 up so I was worried I wouldn't be able to be an operator.  Luckily actual usage proved much smaller.  I am not sure this should even be a requirement but since it does use more bandwidth than a normal node I left it in there - just at 10% of the original value.